app/views/replies/preview.haml

```diff
@@ -1,3 +1,17 @@
+- if @unseen_replies.present?
+  .post-header
+    .padding-15
+      Unseen Replies
+      = link_to post_path(@post, page: 'unread', anchor: 'unread'), target: '_blank' do
+        .view-button Unread (new tab) &raquo;
+      = link_to post_path(@post, page: 'unread', anchor: 'unread') do
+        .view-button.float-left Unread &raquo;
+
+  = render partial: 'replies/single', collection: @unseen_replies, as: :reply
+  - if @unseen_replies.total_pages > 1
+    .post-ender ... and #{@unseen_replies.total_entries - @unseen_replies.per_page} more ...
```

Comment:
It would help if this linked to the rest of the replies. (Yes, there are the above button(s), as well as the permalink buttons, but when there are 5 replies on the page, the top button gets scrolled out of view, so it's less convenient.)